src/main/java/model/Circle.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+public class Circle extends Figure implements Area, Draw {
```

Comment:
```suggestion
public class Circle extends Figure {
```
we can implement these interfaces once in the class Figure
it will be enough